addon/-private/adapters/build-url-mixin.js

```diff
@@ -108,101 +108,258 @@ export default Ember.Mixin.create({
   },
 
   /**
-   * @method urlForFindRecord
-   * @param {String} id
-   * @param {String} modelName
-   * @param {DS.Snapshot} snapshot
-   * @return {String} url
+   Builds a URL for a `store.findRecord(type, id)` call.
+
+   Example:
+
+   ```app/adapters/application.js
```

Comment:
I know, tiny nitpick, but do we want to change this to `app/adapters/user.js` instead?